packages/linux/package.mk

```diff
@@ -135,16 +135,10 @@ post_patch() {
   for f in $PROJECT_DIR/$PROJECT/config/*-overlay.dts; do
     [ -f "$f" ] && cp -v $f $PKG_BUILD/arch/$TARGET_KERNEL_ARCH/boot/dts/overlays
   done
-  if [ -f $PROJECT_DIR/$PROJECT/config/dt-blob.dts ]; then
-    cp -v $PROJECT_DIR/$PROJECT/config/dt-blob.dts $PKG_BUILD/arch/$TARGET_KERNEL_ARCH/boot/dts
-  fi
   if [ -n "$DEVICE" ]; then
     for f in $PROJECT_DIR/$PROJECT/devices/$DEVICE/config/*-overlay.dts; do
```

Comment:
When `$PROJECT_DIR/$PROJECT/devices/$DEVICE/config/*-overlay.dts` doesn't exist (and it doesn't exist at the time of writing) then the next line evaluates to false, which leaves the exit code set to 1 and thus the call to `post_patch()` will fail, meaning the kernel is not built.

adding `exit 0` to the end of `post_patch()` will fix the issue, as will adding `|| true` to any `[ -f "$f" ]` test.

I'll PR a fix shortly.